### ISSUE TITLE
Validation of PEPs should happen in a github action.

### ISSUE THREAD

**Issue by nsheff** [pepkit/pephub.databio.org#3]
We need to create a GitHub action that will validate PEPs when they are submitted to a pephub repository. This action would live in the repository that hosts the PEPs.

> The validation should happen in a github action.

_Originally posted by @nsheff in https://github.com/pepkit/pephub/issues/15#issuecomment-987123031_

Alternatively, if the PEPs will be stored in a database, then the validation should happen at submission to the database.

**Comment by nsheff** [comment]
Did you already implement that, here? https://github.com/pepkit/pephub.databio.org/blob/master/.github/workflows/validate_peps.yml

**Comment by nleroy917** [comment]
Yes. That action **should** work. The script is [here](https://github.com/pepkit/pephub.databio.org/blob/master/scripts/validate_peps.py). It lives at the instance level, however. So, it's really the responsibility of the one hosting the server to validate the PEPs that it hosts.

Maybe I think adding functionality to `pepdb` to validate PEPs on submission makes sense. Then that gets baked into pephub if pephub uses `pepdb`.
